### ISSUE TITLE
No objects found issue when trying to post comments to Icinga 2

### ISSUE THREAD

**Issue by gshashank** [Icinga/logstash-output-icinga#2]
I have created a dummy service in Icinga 2 and trying to send simple comment to that service using the plugin and facing the following issue

`{:host=>"xxxxx", :port=>5665, :path=>"/v1/actions/add-comment?service=%25%7Bhostname%7D%21logstash-comments-service", :body=>"{\"comment\":\"Hello Icinga 2!!!\"}", :error=>#<StandardError: {"error":404.0,"status":"No objects found."}>}`

Below is my configuration file

```
input {
    file {
        path => "/home/demo/logstashinputs1/*"
        start_position => beginning
        ignore_older => 0
    }
}
output {
      icinga {
        host           => "xxxxx" //Remote server
        user           => "xxxxx" //Icinga 2 API user
        password       => "xxxxx" //Icinga 2 Password
        ssl_verify     => false
        action        => "add-comment"
        action_config  => {
         comment => "%{message}"
        }
        icinga_host    => "%{hostname}"
        icinga_service => "logstash-comments-service"
      }
}
```
I have all the permissions for Icinga API user and setting create_object to true gives me the following error

`Unknown setting 'create_object' for icinga`

**Comment by bobapple** [comment]
The log output shows that the field `%{hostname}` is not resolved to an actual string. I think this field is not set by your input. You could verify the fields by using the output `stdout { codec => 'rubydebug' }`

The setting `create_object` is available from `v1.1.0` of this plugin, which version are you using?

**Comment by gshashank** [comment]
@bobapple : Thank you I provided fully qualified host name which resolved the issue :+1:
